tools/experiments/experiments.js

```diff
@@ -273,6 +273,10 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/7670',
   },
   {
+    id: 'amp-gist',
+    name: 'Embed a GitHub gist',
```

Comment:
Please add https://github.com/ampproject/amphtml/issues/8467 as the `cleanupIssue`